src/pybind/mgr/dashboard/run-frontend-unittests.sh

```diff
@@ -45,6 +45,14 @@ fixed, please check the I18N suggestions in 'HACKING.rst':\n"
   failed=true
 fi
 
+# npm resolutions
+npm run fix:audit
+resolutions=`git status | grep package-lock.json`
+if [[ ! -z $resolutions ]]; then
```

Comment:
in this line, would the following equivalent conditional expression be less confusing?

`[[ "$resolutions" ]]` ?

Reply:
Done.